fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -525,6 +529,18 @@ public class SessionVariable implements Serializable, Writable {
     @VariableMgr.VarAttr(name = ENABLE_FUNCTION_PUSHDOWN)
     public boolean enableFunctionPushdown;
 
+    /**
+     * For debugg purpose, dont' merge unique key and agg key when reading data.
+     */
+    @VariableMgr.VarAttr(name = SKIP_STORAGE_ENGINE_MERGE)
+    private boolean skipStorageEngineMerge = false;
```

Comment:
If this is true, can we select __DELETE_SIGN__ directly?

Reply:
updated